cypress/e2e/data-export/cql-filtering-job-profiles.cy.js

```diff
@@ -0,0 +1,66 @@
+import TopMenu from '../../support/fragments/topMenu';
+import FileManager from '../../support/utils/fileManager';
+import GetRandomPostfix from '../../support/utils/stringTools';
+import Permissions from '../../support/dictionary/permissions';
+import Users from '../../support/fragments/users/users';
+import ExportFileHelper from '../../support/fragments/data-export/exportFile';
+import DevTeams from '../../support/dictionary/devTeams';
+import TestTypes from '../../support/dictionary/testTypes';
+import DataExportLogs from '../../support/fragments/data-export/dataExportLogs';
+import SelectJobProfile from '../../support/fragments/data-export/selectJobProfile';
+
+let user;
+
+const validFile = `autoTestValidFile${GetRandomPostfix()}.cql`;
+
+describe('data-export', () => {
+  before('Create test data', () => {
+    cy.createTempUser([
+      Permissions.inventoryAll.gui,
+      Permissions.dataExportAll.gui,
+      Permissions.dataExportEnableModule.gui,
```

Comment:
Test Case only requires "UI: Data export module is enabled" permission

Reply:
Thank you, removed

Reply:
in the permissions.js it's called a bit different: dataExportEnableApp

Reply:
replace

Reply:
Thank you, done